CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 - Fixed #1469 - Broken GitHub location for Firecracker release binary.
 
+### Changed
+
+- Removed `--seccomp-level` command-line argument from jailer parameters
```

Comment:
Ideally, we should try to keep changelog entries concise. Something like:
**Removed redundant `seccomp-level` jailer parameter since it can be simply forwarded to the Firecracker executable.**
*No need to explain how it can be forwarded since users can find that in the doc.*